builders/build/templates/custom-runtime.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Custom Runtime
 description: Any custom runtime can be adapted to be deployed through Tanssi, provided that specific modules are implemented and the required configurations are set.
+icon: material-cog-transfer
```

Comment:
This one being filled in doesn't look totally right. I think we should use this one here instead: `:octicons-gear-24:`